src/router/index.js

```diff
@@ -39,24 +38,24 @@ const router = new Router({
     } else {
       return { x: 0, y: 0 };
     }
-  }
+  },
 });
 
-router.afterEach((to, from) => {
+router.afterEach((to) => { // (to, from)
   // Add a body class specific to the route we're viewing
-  let body = document.querySelector("body");
-  let bodyClasses = body.className.split(" ");
+  let body = document.querySelector('body');
+  // let bodyClasses = body.className.split(' ');
 
-  if (bodyClasses.length > 0) {
-    const newBodyClasses = bodyClasses.filter(theClass =>
-      theClass.startsWith("vue--page--")
-    );
-  }
+  // if (bodyClasses.length > 0) {
```

Comment:
Let's remove this commented out code as well

Reply:
Fixed